pyvista/core/composite.py

```diff
@@ -476,7 +476,7 @@ def get(
         --------
         >>> import pyvista as pv
         >>> from pyvista import examples
-        >>> data = {"poly": pv.PolyData(), "uni": pv.UniformGrid()}
+        >>> data = {"poly": pv.PolyData(), "uni": pv.ImageData()}
```

Comment:
Should we also change the key?
```suggestion
        >>> data = {"poly": pv.PolyData(), "img": pv.ImageData()}
```

Reply:
I'm going to here because you pointed it out, but there are a TON of these and I really don't want to waste time digging all of these up. In time we will transition more of these. 

I tried to change the most obvious ones like `uniform = pv.ImageData()` to `image = pv.ImageData()`

Reply:
Sure thing. I just didn't have the bandwidth to filter what I came across past 1 AM :D